.storybook/addons.js

```diff
@@ -1,4 +1 @@
-import '@storybook/addon-actions/register';
-import '@storybook/addon-links/register';
-import '@storybook/addon-knobs/register';
```

Comment:
I can see the use case for knobs... I think this a point that requires further discussion

Should we create a story for every variation to allow us to explicitly test and demonstrate instead of relying on knobs for example

Reply:
Maybe it can be discussed separately, and leave the import line. I'm ok removing addon-actions, and addon-links can create links between stories... we can add it back if needed

Reply:
Chatted this through with Andy and addon-knobs is going back in 👍